articles/quickstarts/qrng.md

```diff
@@ -48,16 +63,50 @@ We can use this representation to visualize what the code is doing:
 
 <img src="~/media/qrng-H.png" width="450" alt="Preparing a qubit in superposition">
 
-
 * Then we measure the qubit and save the output:
 
 <img src="~/media/qrng-meas.png" width="450" alt="Measuring a qubit and saving the output">
 
 Since the outcome of the measurement is completely random, we have obtained a random bit. We can call this operation several times to create integers. For example, if we call the operation three times to obtain three random bits, we can build random 3-bit numbers (that is, a random number between 0 and 7).
 
-## Creating a complete random number generator using a host program
 
-Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator with a host program.
+## Creating a complete random number generator
```

Comment:
I am not sure this is needed at all for this sample. Since the goal of this sample is just to show a minimal Q# example to get people started, adding a "complete" example may just confuse users.

Reply:
Yeah, I agree, this was one of the conversations we have when creating it, but we wanted to also have an example of how to use the host programs. Maybe we can just add another article for the complete version, and link to it in the "What's next?" link .